catalogue/webapp/components/WorkRedesign/WorkRedesign.js

```diff
@@ -0,0 +1,198 @@
+// $Flow
+import {Fragment, Component} from 'react';
+import Router from 'next/router';
+import NextLink from 'next/link';
+import {font, spacing, classNames} from '@weco/common/utils/classnames';
+import PageLayout from '@weco/common/views/components/PageLayout/PageLayout';
+import InfoBanner from '@weco/common/views/components/InfoBanner/InfoBanner';
+import MetaUnit from '@weco/common/views/components/MetaUnit/MetaUnit2';
+import {workLd} from '@weco/common/utils/json-ld';
+import WorkMedia from '@weco/common/views/components/WorkMedia/WorkMedia';
+import {worksV2Link} from '../../services/catalogue/links';
+import Layout10 from '@weco/common/views/components/Layout10/Layout10';
+import Layout12 from '@weco/common/views/components/Layout12/Layout12';
+import SearchBox from '@weco/common/views/components/SearchBox/SearchBoxV2';
+import SpacingSection from '@weco/common/views/components/SpacingSection/SpacingSection';
+
+type Work = Object;
+type Props = {|
+  work: Work,
+  iiifImageLocationUrl: ?string,
+  licenseInfo: ?string,
+  iiifImageLocationCredit: ?string,
+  iiifImageLocationLicenseId: ?string
+|}
+
+class WorkRedesign extends Component<Props> {
+  state = {
+    query: ''
+  }
+
+  handleSubmit = (event) => {
```

Comment:
Any reason we did this instead of hooks?

Reply:
Dropped a couple of hours on hooks with @gestchild but behaviour seemed buggy (the input would only update one character at a time with `onInput` without reselecting the input field. Possible that the error was elsewhere with something we'd done, but couldn't find it so opted for the old-skool tried-and-true method for the time-being.

Reply:
> tried-and-true method 

Hooks are just an abstraction on that - might be good to pair once this is in to have a go and get a good understanding of how it works.

Reply:
yeah, not suggesting we _don't_ move to using hooks, but didn't see the sense in wasting more time on it.